src/main/resources/archetype-resources/silo/src/main/resources/application-production.yml

```diff
@@ -1,22 +1,20 @@
+server:
+    port: 0
+
 spring:
     datasource:
-        # TODO: dont forget to CHANGE the url
-        url: jdbc:MYSQL_OR_POSTGRES_OR...
-        poolSize: 20
+      url: jdbc:CHANGEME
+      host: CHANGEME
+      port: CHANGEME
+      username: CHANGEME
+      password: CHANGEME
 
 clients:
     customerClient:
-        host: http://customer-silo.proxy.dc1.rebuy
+        host: customer-silo.proxy.dc1.rebuy
 
     permissionClient:
         host: customer-silo.proxy.dc1.rebuy
 
-permissions:
```

Comment:
das cached die Berechtigungen, welche in der customer.permissions stehen. Dies wird benutzt um fest zu stellen, ob ein user berechtigt ist auf einen Endpunkt zuzugreifen. Diese Berechtigungen sind eigentlich statisch und ändern sich kaum. Von daher kann man da ruhig hohe werte benutzen falls Änderungen unbedingt sofort greifen sollen muss man halt das Silo neustarten.